### PR TITLE
Add FuseBox

### PR DESCRIPTION
**Pull request by aretecode**
adding fusebox to the build / bundling tools:
> [FuseBox](https://github.com/fuse-box/fuse-box) - Extremely fast build system that combines the power of webpack, JSPM and SystemJS, with first-class typescript support.

### CODE REVIEW

**Comment by itsjtwright** [comment]
honestly a really cool item, think you need a description for your pr and you are set

**Comment by aretecode** [comment]
@itsjtwright how's that?

**Comment by itsjtwright** [comment]
I think it should be, mr @sindresorhus takes his time on some of this stuff though, just gotta wait sometimes.

File: readme.md [hunk 1]
```diff
@@ -295,7 +295,7 @@ Check out my [blog](https://blog.sindresorhus.com) 🦄 or say *hi* on [Twitter]
 - [ygor](https://github.com/shannonmoeller/ygor) - Promising task runner for when `npm run` isn't enough and everything else is too much.
 - [grunt](http://gruntjs.com) - Task runner that can perform repetitive tasks like minification, compilation, unit testing, linting, etc.
 - [Fly](https://github.com/bucaran/fly) - Modern build system based in co-routines, generators and promises.
-
+- [FuseBox](https://github.com/fuse-box/fuse-box) - Extremely fast build system that combines the power of webpack, JSPM and SystemJS, with first-class typescript support.
```
Comment (SamVerschueren) [inline readme.md]: `TypeScript`

**Comment by aretecode** [comment]
@SamVerschueren updated

**Pull request merged**